src/tests/simple_tests/README.md

```diff
@@ -0,0 +1,5 @@
+In tests in `.asm` files it is possible to use several additional directives (not stable):
+- `print("<TEXT>")` - print text `<TEXT>` in console. Max length of text is 30 symbols
+- `print(<src>)` - print value of `<src>` (register/constant/etc) in console
```

Comment:
what would be nice (in future) - is to have the print with 2 arguments (one string, and one register etc).

this way you can do stuff like:
print("before call", r0)
etc.

(otherwise, once the test is bigger, having many prints can be harder to manage).

But as I've mentioned - in separate PR